README.org

```diff
@@ -265,6 +265,20 @@ Or for use-package users
   :hook (dashboard-before-initialize . solaire-mode)
 #+end_src
 
+To use it with [[https://github.com/hlissner/emacs-hide-mode-line][hide-mode-line]]
+#+begin_src elisp
+  (add-hook 'dashboard-mode #'hide-mode-line-mode)
+  ;; Or for use-package
+  (use-package hide-mode-line
+    :hook (dashboard-mode . hide-mode-line-mode)
+    ...)
+#+end_src
+
+Or for doom-modeline users
+#+begin_src elisp
+  (add-to-list 'doom-modeline-mode-alist '(dashboard-mode))
+#+end_src
```

Comment:
It's a good time to place all these in the FAQ section. 🤔 

Maybe a FAQ.org.

Reply:
Added